From: Michal Malek <michalm@jabster.pl>
Date: Sun, 01 May 2011 10:27:12 +0000
Subject: Set error status when CDDB query fails. Without it the error is buing not reported and as a result all entries in audio project are cleared up.
X-Git-Url: http://quickgit.kde.org/?p=k3b.git&a=commitdiff&h=fb10bb8fc296335488472468be107b8c4cdb9251
---
Set error status when CDDB query fails. Without it the error is buing not reported and as a result all entries in audio project are cleared up.
---


--- a/libk3b/tools/k3bcddb.cpp
+++ b/libk3b/tools/k3bcddb.cpp
@@ -116,6 +116,7 @@
         }
     }
     else {
+        q->setError( KJob::UserDefinedError );
         q->setErrorText( KCDDB::resultToString( result ) );
     }
 

